blackboxtest/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM circleci/openjdk:jdk-node-browsers
+
+WORKDIR /home/circleci
+COPY package.json yarn.lock nightwatch-deps.sh /home/circleci/
+RUN yarn install --non-interactive --frozen-lockfile
+RUN curl -s https://selenium-release.storage.googleapis.com/3.9/selenium-server-standalone-3.9.1.jar > /home/circleci/selenium-server-standalone.jar
+RUN curl -s https://chromedriver.storage.googleapis.com/2.36/chromedriver_linux64.zip | funzip > /home/circleci/chromedriver && chmod +x /home/circleci/chromedriver
```

Comment:
Isn't there a selenium image we can use directly?

Reply:
there aren't any circleci-provided ones but yes there probably is one out there; i found this way to work pretty well & it gets cached anyways. now we have our own selenium image ;)

Reply:
Looks like there is an official selenium image, if that's a drop in replacement I suggest using that just for the free updates :)

https://hub.docker.com/r/selenium/standalone-chrome